docs/content/Deployment/Deployment-Guide.md

```diff
@@ -411,3 +412,25 @@ git push heroku master
 
 For more details, take a look at the
 [official documentation from Heroku](https://devcenter.heroku.com/articles/build-docker-images-heroku-yml).
+
+## Cube Cloud
+
+[[info |]]
+| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
+| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
```

Comment:
```suggestion
<!-- prettier-ignore-start -->
[[info |]]
| [Cube Cloud][link-cube-cloud] currently is in early access. If you don't have
| an account yet, you can [sign up to the waitlist here][link-cube-cloud].
 <!-- prettier-ignore-end -->
```

Reply:
@keydunov Should the waitlist link be different to the main Cube Cloud one?

Reply:
We have only one page so far.